<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="death">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="restart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RESTART</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>141</width>
      <height>94</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="whatsThis">
        <string>15</string>
       </property>
       <property name="text">
        <string>Лёгкий</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">level_btn</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="whatsThis">
        <string>12</string>
       </property>
       <property name="text">
        <string>Средний</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">level_btn</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="whatsThis">
        <string>8</string>
       </property>
       <property name="text">
        <string>Сложный</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">level_btn</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>true</underline>
        </font>
       </property>
       <property name="whatsThis">
        <string>6</string>
       </property>
       <property name="text">
        <string>Непроходимый</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">level_btn</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Уровень сложности</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Статус игры:</string>
    </property>
   </widget>
   <widget class="QLabel" name="game_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите букву!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Подсказка к слову:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="help_window">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>141</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_7">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="whatsThis">
        <string>3</string>
       </property>
       <property name="text">
        <string>Три игрока</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">cnt_players</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_5">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="whatsThis">
        <string>1</string>
       </property>
       <property name="text">
        <string>Один игрок</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">cnt_players</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_6">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="whatsThis">
        <string>2</string>
       </property>
       <property name="text">
        <string>Два игрока</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">cnt_players</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButton_8">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="whatsThis">
        <string>4</string>
       </property>
       <property name="text">
        <string>Четыре игрока</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">cnt_players</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Количество игроков</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_add_word">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>10</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добавить новое слово</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="cnt_players"/>
  <buttongroup name="level_btn"/>
 </buttongroups>
</ui>
